src/coprocessor/dag/expr/builtin_string.rs

```diff
@@ -3259,4 +3293,26 @@ mod tests {
             assert_eq!(got, exp);
         }
     }
+
+    #[test]
+    fn test_instr() {
+        let cases: Vec<(&str, &str, i64)> = vec![
+            ("f", "foobArbar", 1),
+            ("o", "foobArbar", 2),
+            ("0", "foobArbar", 0),
+            ("A", "foobArbar", 5),
+            ("r", "foobArbar", 6),
+            (" ", "foobArbar", 0),
+            ("", "", 0),
+            ("BaR", "foobArbar", 0),
+            ("bar", "foobArbar", 7),
+            ("好世", "你好世界", 4),
```

Comment:
I think it should return `2` here

Reply:
I have run the test and passed。It's return 4. I think it's something about encoding.